tests/components/lg_soundbar/test_config_flow.py

```diff
@@ -22,13 +26,21 @@ def create_temescal_response(msg: str, data: Optional[dict] = None):
         return response
 
     def temescal_side_effect(addr, port, callback):
-        instance.get_mac_info.side_effect = lambda: callback(
+        def invoke_callback(response):
+            def run_thread():
+                sleep(delay)
+                callback(response)
+
+            thread = threading.Thread(target=run_thread, daemon=True)
```

Comment:
I think we can use `hass.add_job` to schedule the whole `invoke_callback` to the thread pool. Then we don't need a specific thread here.